login.md

```diff
@@ -1,27 +1,41 @@
-# Getting Started in UNIX Coding Environment
+# Getting Started in the UNIX Coding Environment
 
-Welcome to COMSW 3157 Advanced Programming with Dr. Brian Borowski! Prior to this course, you may have used various coding environments like Codio, VSCode, etc. But this guide will introduce you to the one that we use for AP. It is very important that you read through this guide so that you will be able to do the assignments for this course. 
+Welcome to COMSW 3157 Advanced Programming with Dr. Brian Borowski! Prior to this course, you may have used various coding environments like Codio, VSCode, etc. But this guide will introduce you to the one that we use for AP, the UNIX Coding Environment on our remote class server. It is very important that you read through this guide so that you will be able to complete the assignments for this course. 
 
 You will be interacting with the UNIX coding environment via the command-line. 
 
 ## What is the "Command-Line"?
-The command-line in a UNIX coding environment allows you to interact with the operating system and execute commands. From the command line, you can navigate file systems, run programs, amongst other things. Think of this as a very basic way to interact with an operating system and an alternative to a mouse/trackpad. 
+The command-line in a UNIX coding environment allows you to interact with the operating system and execute commands. From the command line, you can navigate file systems, run programs, amongst other things. Think of this as a very simplified way of interacting with an operating system or an alternative to a mouse/trackpad. 
```

Comment:
I think "basic" works better than "simplified" and I see why you changed the "or" to the "and" which sounds better but its less correct.